### PR TITLE
Disable trusted exepath

### PR DESCRIPTION
**Pull request by erthalion**
## Description

Vanilla Falco has introduced more rigorous exepath logic which resolves
symlinks when needed. By itself it's a welcomed improvement, but turns
out it upsets ACS builtin policies that often rely on the original
exepath implementation. Temporary disable trusted exepath until we will
figure out how to migrate policies to use this information.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Updated documentation accordingly

**Automated testing**
  - [ ] Added unit tests
  - [ ] Added integration tests
  - [ ] Added regression tests

## Testing Performed

Local testing with an image triggering violations.

### CODE REVIEW

File: kernel-modules/MODULE_VERSION [hunk 1]
```diff
@@ -1 +1 @@
-2.9.0-rc2
+2.9.0-rc4
```
Comment (Stringy) [inline kernel-modules/MODULE_VERSION:1]: why do we need a module version bump for this?

File: builder/Dockerfile [hunk 2]
```diff
@@ -15,7 +15,7 @@ RUN dnf -y update \
         binutils-devel \
         bison \
         ca-certificates \
-        clang-15.0.7 \
+        clang-17.0.6 \
```
Comment (Stringy) [inline builder/Dockerfile:18]: since this is fixed in master now I think we need to rebase

File: kernel-modules/MODULE_VERSION (hunk 1 above)
Comment (erthalion) [reply kernel-modules/MODULE_VERSION:1]: I was under the impression it wont pickup Falco changes otherwise. You're saying it's not correct?

Comment (Stringy) [reply kernel-modules/MODULE_VERSION]: a bump is only necessary if there is a change to the drivers, so since this is just userspace we don't need to :)

**Review by Stringy — APPROVED** [review]

**Pull request merged**